auth_manager.go

```diff
@@ -89,6 +103,56 @@ func (b *DefaultSessionManager) Init(store storage.Handler) {
 	b.disableCacheSessionState = config.Global().LocalSessionCache.DisableCacheSessionState
 	b.store = store
 	b.store.Connect()
+
+	if b.asyncWrites {
+		//check pool size in config and set to 50 if unset
+		b.poolSize = config.Global().SessionUpdatePoolSize
+		if b.poolSize <= 0 {
+			b.poolSize = 50
+		}
+
+		log.WithField("Auth Manager", b.poolSize).Debug("Session update async pool size")
+
+		b.updateChan = make(chan *SessionUpdate)
```

Comment:
I think we need to have that channel buffered with buffer size specified in config

Reply:
What is a sensible default for this?

Reply:
Using 1000 as default